supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+[[partner-and-hybrid-cloud-infrastructure-conventions]]
+
+[discrete]
+[[access-mode]]
+==== image:images/yes.png[yes] access mode (noun)
+*Description*: The concurrency method used when accessing a storage component, such as `readwriteonce` or `readwritemany`.
+
+*Use it*: Yes
+
+
+[discrete]
+[[availability]]
+==== image:images/caution.png[with caution] availability (adjective)
+The state of being available to perform work in a system, in the context of high availability or availability zones. The ability of a component to be utilized at a given time.
+
+Ensure that availability is not erroneously used to indicate that a server or disk is online or available, e.g. do not say "the server has availability".
+
+*Use it*: With caution
+
+*See also*: xref:high-availability[high availability], xref:availability-zone[availability zone]
+
+
+[discrete]
+[[availability-zone]]
+==== image:images/yes.png[yes] availability zone (noun)
+*Description*: A logical context or geographical region used to provide high availability. This grouping can be defined by a technology, or according to unique use cases.
+
+Ensure that availability is not erroneously used to indicate that a server or disk is online or available, e.g. do not say "the server has availability".
+
+*Use it*: Yes
+
+*See also*: xref:high-availability[high availability]
+
+[discrete]
+[[back-end]]
+==== image:images/caution.png[with caution] back end (noun)
+*Description*: A back end is the location for stored data. Hardware and software comprise a storage back end. Hyphenate when used as an adjective, for example, back-end storage.
+
+*Use it*: With caution, it is better to refer to "back-end storage" than to "the back end" as this is ambiguous even in a storage context.
+
+*Incorrect forms*: backend
+
+
+[discrete]
+[[back-up]]
+==== image:images/yes.png[yes] back up (verb)
+*Description*: Create a copy of data, configuration files, etc. to reduce risk of data loss. "Backup" when used as a noun, for example, the backup of your data is the data that was backed up.
+
+*Use it*: Yes
+
+*Incorrect forms*: back-up
+
+
+[discrete]
+[[bare-metal-service]]
+==== image:images/yes.png[yes] Bare Metal service (noun)
+*Description*: The Bare Metal service is a Red Hat OpenStack Platform product that provisions bare metal machines.
+
+*Use it*: Yes
+
+
+[discrete]
+[[bind-v]]
+==== image:images/yes.png[yes] bind (verb)
+*Description*: To associate two or more objects in a particular scope, for example, in OpenShift Container Storage, when a Persistent Volume Claim is created, a Persistent Volume that matches the claim's requirements is bound to the Persistent Volume Claim.
```

Comment:
Can you lowercase the instances of "Persistent Volume" and "persistent volume claim"?